### PR TITLE
Xunit template

### PR DESCRIPTION
**Pull request by livarcocc**
@piotrpMSFT @eerhardt @jgoshi @krwq @codito

### CODE REVIEW

**Comment by livarcocc** [comment]
Note that the output for this is still not ideal. But I still want to go ahead and get the template in.

Microsoft (R) Test Execution Command Line Tool Version 15.0.0.0
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...

File: src/dotnet/commands/dotnet-new/Program.cs [hunk 1]
```diff
@@ -137,7 +137,8 @@ public static int Run(string[] args)
                                { 
                                    new { Name = "Console", isMsBuild = true }, 
                                    new { Name = "Web", isMsBuild = true }, 
-                                   new { Name = "Lib", isMsBuild = true }
+                                   new { Name = "Lib", isMsBuild = true },
+                                   new { Name = "Xunittest", isMsBuild = true }
```
Comment (eerhardt) [inline src/dotnet/commands/dotnet-new/Program.cs]: Do we have a test for the template?

File: src/dotnet/commands/dotnet-new/Program.cs [hunk 2]
```diff
@@ -137,7 +137,8 @@ public static int Run(string[] args)
                                { 
                                    new { Name = "Console", isMsBuild = true }, 
                                    new { Name = "Web", isMsBuild = true }, 
-                                   new { Name = "Lib", isMsBuild = true }
+                                   new { Name = "Lib", isMsBuild = true },
```
Comment (eerhardt) [inline src/dotnet/commands/dotnet-new/Program.cs]: Can we get rid of the "isMSBuild" now?  Shouldn't all templates be MSBuild now?

**Comment by livarcocc** [comment]
@dotnet-bot Test this please

**Comment by livarcocc** [comment]
@dotnet-bot Test OpenSUSE13.2 x64 Debug Build
@dotnet-bot Test Ubuntu x64 Release Build

**Comment by livarcocc** [comment]
@dotnet-bot Test OSX x64 Release Build

**Comment by livarcocc** [comment]
@dotnet-bot Test OSX x64 Release Build

**Pull request merged**